user-guide/writing-tests.md

```diff
@@ -144,6 +144,22 @@ context "Some Context" do
 end
 ```
 
+## Details
+
+When tests fail, it is often necessary to see details of the test scenario itself in order to diagnose the failure. However, it is generally undesirable to see information about the test scenario when reading the output from a test file that passes. For that reason, _detailed_ output can be printed with `detail`:
+
+```ruby
+context "Some Context" do
+  test "Passing test" do
+    detail "Will not be printed"
```

Comment:
This example would seem to affirm that test blocks cannot have assertions in them, which is an error.

Reply:
Corrected as well, thanks.